src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -0,0 +1,428 @@
+        if (this.group && this.group != this.groupPeople) {
+          //filter people in group against search term
+          peoples = await client.getPeopleFromGroup(this.group);
+          for (let person of peoples) {
+            // set image to @ to flag the mgt-person component to
+            // query the image from the graph
+            person.image = '@';
+          }
+
+          peoples = peoples.filter(function(person) {
+            return person.displayName.toLowerCase().indexOf(name) !== -1;
+          });
+          if (peoples) {
+            this.filterPeople(peoples);
+            this.groupPeople = this.group;
+          } else {
+            this.people = [];
+            this.filterPeople(peoples);
+          }
+        } else {
+          if (this.group) {
+            let oldPeople: any = this.people;
+            peoples = oldPeople.filter(function(person) {
+              return person.displayName.toLowerCase().indexOf(name) !== -1;
+            });
+            for (let person of peoples) {
+              // set image to @ to flag the mgt-person component to
+              // query the image from the graph
+              person.image = '@';
+            }
+            this.filterPeople(peoples);
+          }
+          peoples = await client.findPerson(name);
+          for (let person of peoples) {
+            // set image to @ to flag the mgt-person component to
+            // query the image from the graph
+            person.image = '@';
+          }
+          if (peoples) {
+            peoples = peoples.filter(function(person) {
+              return person.displayName.toLowerCase().indexOf(name) !== -1;
+            });
+            this.filterPeople(peoples);
+          } else {
+            this.people = [];
+            this.filterPeople(peoples);
+          }
+        }
+      }
+    }
+  }
+
+  private filterPeople(peoples: any) {
+    //check if people need to be updated
+    //ensuring people list is displayed
+    //find ids from selected people
+    if (peoples) {
+      let id_filter = this._selectedPeople.map(function(el) {
+        return el.id;
+      });
+      //filter id's
+      let filtered = peoples.filter(function(person) {
+        return id_filter.indexOf(person.id) === -1;
+      });
+      if (filtered.length == 0 && this._userInput.length > 0) {
+        this.people = [];
+        return;
+      } else {
+        if (filtered.length) {
+          filtered[0].isSelected = 'fill';
+          this.arrowSelectionCount = 0;
+          this.people = filtered;
+        }
+      }
+    }
+    this.isLoading = false;
+  }
+
+  private removePerson(person: MgtPersonDetails) {
+    let chosenPerson: any = person;
+    let filteredPersonArr = this._selectedPeople.filter(function(person) {
+      return person.id !== chosenPerson.id;
+    });
+    this._selectedPeople = filteredPersonArr;
+    this.fireCustomEvent('selectionChanged', this._selectedPeople);
+    this.renderChosenPeople();
```

Comment:
remove this

Reply:
Removed: 
https://github.com/microsoftgraph/microsoft-graph-toolkit/commit/0d22ec6a8da3c2cb53c3f0b927ff429f706d229c#diff-4b7ee5f7ffb7a1c981093bdb2f2b146fL264